tests/unittests/tests-cncbor/tests-cncbor.c

```diff
@@ -0,0 +1,191 @@
+﻿/*
+ * Copyright (C) Lorenz Hüther, Mathias Detmers
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+ /**
+  * @ingroup    tests
+  * @{
+  *
+  * @file
+  * @brief      Unit tests for pkg cn-cbor.
+  *
+  * @author     Lorenz Hüther
+  * @author     Mathias Detmers
+  */
+
+#define EBUF_SIZE 32
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "cn-cbor/cn-cbor.h"
+#include "embUnit.h"
+
+typedef struct buffer {
+    size_t size;
+    unsigned char *pntr;
+} buffer_t;
+
+typedef struct cbor_failure
+{
```

Comment:
opening brace goes one line up, like done for buffr_t.

Reply:
Will be fixed!